cvat/apps/engine/serializers.py

```diff
@@ -391,11 +392,25 @@ def validate_chunk_size(self, value):
             raise serializers.ValidationError('Chunk size must be a positive integer')
         return value
 
+    def validate_pattern(self, value):
+        import string
```

Comment:
What is the point of importing the package here? How it is better importing in the file beginning?

Reply:
I forgot to move it, thanks